R/CellGraph.R

```diff
@@ -55,8 +55,8 @@ CellGraph <- setClass(
 #'
 #' edge_list <-
 #' ReadMPX_item(
-#'   system.file("extdata/PBMC_10_cells",
-#'               "Sample01_test.pxl",
+#'   system.file("extdata/mock_data",
```

Comment:
Any possibility we can access the data by e.g. `data(fivempxcells)`? I realize this is not within the scope of this PR, but for future perspectives

Reply:
Yeah I think we'd have to create an rds object stored separately to get that. So the total file size would double. I think it could still be ok though.